modules/mysql/main.tf

```diff
@@ -192,7 +192,7 @@ resource "google_sql_user" "additional_users" {
   for_each = local.users
   project  = var.project_id
   name     = each.value.name
-  password = lookup(each.value, "password", random_password.user-password.result)
+  password = lookup(each.value, "password", random_password.additional_passwords[each.key].result)
```

Comment:
Can you add a section to the upgrade guide for existing users? I suspect users who already applied can set the value of`random_password.user-password.result` to var. additional_users to prevent a diff.
https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/docs/upgrading_to_sql_db_12.0.0.md

Reply:
@carash - we'd like this to part of the upcoming release. Let us know if you can update this in the next 24h. Thank you!

Reply:
will do, I'll update the PR today along with the updated tests

Reply:
I suspect an upgrade guide to prevent a diff might require manipulating the tfstate file directly, is that fine?

Reply:
@carash I was thinking since they have already applied it with `random_password.user-password.result`, they can retrieve output from `generated_user_password` and use that to construct var. additional_users like 
```diff
+ data "google_secret_manager_secret_version" "password" {
+  secret_id = "pg-user-pass"
+  project   = var.project_id
+ }
...
additional_users = [
     {
       name     = "user1"
+       password = data.google_secret_manager_secret_version.password.secret_data
     }
   ]
```

similar to what did here https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/docs/upgrading_to_sql_db_11.0.0.md#switched-to-using-random_password-to-generate-default-passwords

Reply:
ah I see. that'll do it, thanks